lib/components/map/elevation-point-marker.js

```diff
@@ -0,0 +1,45 @@
+import coreUtils from '@opentripplanner/core-utils'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { CircleMarker } from 'react-leaflet'
+
+class ElevationPointMarker extends Component {
```

Comment:
Let's give the class some jsdoc about how this works/what it's showing/how to turn it on.